### PR TITLE
Added & Styled view pages

### PR DESCRIPTION
**Pull request by Shakir-Hussain12**
**For this PR, I :**
- Added html code to view files.
- Used embedded ruby where required.
- Created partials for generating different sections of the page.
- Created a helper for comments section.
- Styled the Elements according to the wireframe.

### CODE REVIEW

File: app/views/posts/_comlist.html.erb [hunk 1]
```diff
@@ -0,0 +1,7 @@
+<% @posts.each do |post| %>
+    <%= render partial:'users/show', object:post, as: :post %>
+    <div class="comment-container">
+        <h3>Post Comments</h3>
+        <%= show_comments(post) %>
+    </div>
```
Comment (elisha2kyakpo1) [inline app/views/posts/_comlist.html.erb]: - [x] Kindly make use of the model functions(methods) you created previously to render the 5 most recent comments, we do not have to render all the comments, only the most recent ones.

model

File: app/views/users/_show.html.erb [hunk 2]
```diff
@@ -0,0 +1,7 @@
+<li class="post-item">
+    <h3>Post#<%= post.id %></h3>
+    <div class="inner-items">
+        <p><%= post.text %></p>
+        <p class="item-no">Comments: <%= post.comments_count == nil ? 0 : post.comments_count %>, Likes: <%= post.likes_count == nil ? 0 : post.likes_count %></p>
```
Comment (elisha2kyakpo1) [inline app/views/users/_show.html.erb]: - [x] Let's use the most recent posts model method you created in the previous milestone, this method should render only the three most recent posts for a user.

File: app/views/posts/show.html.erb [hunk 3]
```diff
@@ -1,3 +1,8 @@
...
-</h1>
\ No newline at end of file
+<div class="top-com">
+    <h1>Post#<%= @post.id %> by <%= @user.name %></h1>
+    <%= render partial: 'users/show', object: @post, as: :post %>
+</div>
+<div class="comment-container">
+    <h3>Post Comments</h3>
+    <%= show_comments(@post) %>
+</div>
```
Comment (elisha2kyakpo1) [inline app/views/posts/show.html.erb:8]: - [x] We should have a button to see all the posts on the user's show page. This button should take users to all the posts for that user(for example, the first user in this project)

recent

**Review by elisha2kyakpo1 — CHANGES_REQUESTED** [review]
Hi 👋 @Shakir-Hussain12, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights

✔️ The PR title and description are good and professional.
✔️ No linter errors.
✔️ Descriptive readme file.

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you consider them as they can improve your code._

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by MeqdamAlqudah — APPROVED** [review]
Hi @Shakir-Hussain12  

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
image

## Highlights 
:+1: most recent posts functionality 
:+1: most recent comments functionality 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**